firenoc/firenoc-services/src/utils/index.js

```
@@ -182,17 +182,35 @@ export const getUpdatedTopic = (tenantId, topic) => {
+      const parts = tenantId.split('.');
+      schemaName = parts[stateSchemaIndexPositionInTenantId];
+    } else {
+      schemaName = tenantId;
+    }
+
+    // Case-insensitive global replace
+    const pattern = new RegExp(SCHEMA_REPLACE_STRING, 'gi');
+    finalQuery = query.replace(pattern, schemaName);
+  } else {
+    // Remove `{schema}.` case-insensitively
+    const pattern = new RegExp(SCHEMA_REPLACE_STRING + '\\.', 'gi');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)